keyboards/dz60/keymaps/jarred/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_USER_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
@drashna Actually, I think this file can be deleted.

- Line 4 isn't needed; it's included automatically by the build script.
- Line 6 isn't needed. It's default behavior.

With your changes, that leaves only `#pragma once` in this file.